cmd/chainbridge/deploy.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"os"
+	"os/exec"
+	"strings"
+
+	log "github.com/ChainSafe/log15"
+	"github.com/urfave/cli"
+)
+
+var deployCommand = cli.Command{
+	Action:   handleDeployCmd,
+	Name:     "testdeploy",
```

Comment:
this should be called `deploy` it should also sit under `eth`

i.e - `chainbridge eth deploy <flags>`

Reply:
It will probably be migrated out of `chainbridge` altogether, as mentioned in #193. For now, should I comment this out or just leave it as part of `chainbridge` for this ticket?
@GregTheGreek  @ansermino

Reply:
Right but #193 still requires this to be a deploy sub-command. So I would still re-name this

Reply:
That make sense, renamed.